### ISSUE TITLE
Allow using .yaml file extension instead of .yml

### ISSUE THREAD

**Issue by logdnalf** [deepmind/kapitan#505]
**Describe the bug/feature**
I need that Kapitan can produce YAML files from Jsonnet which have a `.yaml` file extension instead of `.yml`.

**To Reproduce**
`kapitan compile` produces `.yml` files.

**Expected behavior**
Allow Kapitan to produce YAML files with a `.yaml` file extension, likely controlled by the `output_type` in the inventory. For example, `output_type: yaml` would produce files named `*.yaml` while `output_type: yml` would produce files named `*.yml`.

**Comment by uberspot** [comment]
This seems to be a duplicate of https://github.com/deepmind/kapitan/issues/276 and that is blocked by https://github.com/salt-formulas/reclass/pull/87 getting merged.

**Comment by adrianchifor** [comment]
I think #276 is about reclass and this is more about how kapitan outputs files.

**Comment by Cyanopus** [comment]
This seems to be an easy task. 

https://github.com/deepmind/kapitan/blob/890094592f9d61844398828958339a29ef764890/kapitan/inputs/jsonnet.py#L77-L79
to liberal
```python
elif output in ["yml", "yaml"]:
                file_path = os.path.join(compile_path, "%s.%s" % (item_key, output))
```
@uberspot @adrianchifor Good for PR?

**Comment by adrianchifor** [comment]
@Cyanopus Yeah that should do
